### DISCUSSION TITLE
add css for value in graph

### DISCUSSION THREAD

**Discussion by workingJoey** [Budibase/budibase#14011] [Help] [answered]
How do i change the font color of the value in a graph in budibase version 2.8

![image](https://github.com/Budibase/budibase/assets/173664546/1d82cf33-6b4e-4cc7-884c-8e8261596c1f)

**Comment by ConorWebb96** [comment, marked as answer]
You might be able to use inspect element to get the class of that element shown in the screenshot and then use the embed component with `<style></style>` tags and css to override the styling.

**Reply by workingJoey** [reply]
i solve it by adding the embed as instructed with the the css

`<style>
 div.svelte-i4ux9u .apexcharts-datalabel {
fill: #2F2F2F;
 }
</style>`
